src/webapp/application.py

```diff
@@ -0,0 +1,119 @@
+import json
+
+import matplotlib
+import matplotlib.pyplot as plt
+import boto3
+import pandas as pd
+import numpy as np
+import datetime
+import requests
+import csv
+
+from flask import Flask, render_template, request
```

Comment:
generally you want to organize imports like this:
```
# Standard library files first. Alphabetical order.
import datetime 
import csv
import json

# Packages installed with pip go here after a space. Alphabetical order.
import boto3
from flask import Flask

# Your own modules. Alphabetical order.
```

Reply:
sure!